HybridCamLib/src/cam/CamView+Action+Core.swift

```diff
@@ -50,7 +45,12 @@ extension CamView {
             onVideoCaptureComplete(nil, error)
          }
       }
-      connection.isVideoMirrored = (connection.isVideoMirroringSupported && device.position == .front && mirrorFrontCamera)
+      guard videoOutput.isRecording == false else { onVideoCaptureComplete(nil, CaptureError.alreadyRecording); return }
+      guard let connection: AVCaptureConnection = videoOutput.connection(with: .video) else { onVideoCaptureComplete(nil, CaptureError.noVideoConnection); return }
+      if  {
```

Comment:
If what? 😅

Reply:
Sorry, I don't know why it doesn't say `if connection.isVideoOrientationSupported {` anymore. Didn't mean to delete that, my bad 😅